docs/understand/roadmap.md

```diff
@@ -18,101 +18,99 @@ redirect_from: ../roadmap.html
 
 ### Snowflake Support: External tables on lakeFS <span>Requires Discussion</span>{: .label .label-yellow }
 
-Since Snowflake supports reading external tables from an object store, we'd like to extend this support to work with lakeFS repositories that are hosted on top of [supported object stores](https://docs.snowflake.com/en/user-guide/data-load-snowpipe-intro.html#supported-cloud-storage-services){: target="_blank" }.
-This could be done by utilizing Snowflake's support for `SymlinkInputFormat` similar to how [Delta Lake support is implemented](https://docs.databricks.com/delta/snowflake-integration.html){: target="_blank" }, and later on by having a native integration with Snowflake itself. If you'd like to hear more:
-
-[Contact us, we'd love to talk about it!](mailto:hello@treeverse.io?subject=using+lakeFS+with+Snowflake){: target="_blank" class="btn" }
+Since Snowflake supports reading external tables from an object store, we'd like to extend this support to work with lakeFS repositories hosted on top of the [supported object stores](https://docs.snowflake.com/en/user-guide/data-load-snowpipe-intro.html#supported-cloud-storage-services){: target="_blank" }.
+This could be done by utilizing Snowflake's support for `SymlinkInputFormat` similarly to how [Delta Lake support is implemented](https://docs.databricks.com/delta/snowflake-integration.html){: target="_blank" } and later on by having a native integration with Snowflake itself. If you'd like to hear more, [contact us, we'd love to talk about it some more!](mailto:hello@treeverse.io?subject=using+lakeFS+with+Snowflake){: target="_blank" class="btn" }
 
 
 ### Pluggable diff/merge operators
 
 Currently, lakeFS supports merging and comparing references by doing an object-wise comparison. 
-For unstructured data and some forms of tabluar data (namely Hive structured tables), this works fine. 
+For unstructured data and some forms of tabluar data (namely, Hive structured tables), this works fine. 
 
-However, in some cases, simply creating a union of object modifications from both references isn't good enough. 
-Modern table formats such as Delta Lake, Hudi and Iceberg rely on a set of manifest or log files that describe the logical structure of the table. 
-In those cases, a merge operation might have to be aware of the structure of the data: 
+However, in some cases simply creating a union of object modifications from both references isn't good enough. 
+Modern table formats such as Delta Lake, Hudi, and Iceberg rely on a set of manifest or log files that describe the logical structure of the table. 
+In those cases, a merge operation might have to be aware of the data structure: 
```

Comment:
Data structure is an overloaded term - "the structure of the data" makes more sense here.

Reply:
Fixed